packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -775,22 +799,39 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
                                     </Tooltip>
 
                                     {userCanManageExplore && chartPathname && (
-                                        <Menu.Item
-                                            icon={
-                                                <MantineIcon
-                                                    icon={IconTelescope}
-                                                />
+                                        <Tooltip
+                                            label={
+                                                'This chart contains custom dimensions, you will not be able to run custom SQL on explore.'
                                             }
-                                            disabled={isEditMode}
-                                            onClick={() =>
-                                                handleCreateShareUrl(
-                                                    chartPathname,
-                                                    chartSearch,
-                                                )
+                                            position="top-start"
+                                            variant="xs"
+                                            disabled={
+                                                !cannotUseCustomDimensions
                                             }
                                         >
-                                            Explore from here
-                                        </Menu.Item>
+                                            <Menu.Item
+                                                icon={
+                                                    <MantineIcon
+                                                        icon={IconTelescope}
+                                                    />
+                                                }
+                                                disabled={isEditMode}
+                                                onClick={() =>
+                                                    handleCreateShareUrl(
+                                                        chartPathname,
+                                                        chartSearch,
+                                                    )
+                                                }
+                                            >
+                                                <Group>
+                                                    Explore from here
+                                                    <MantineIcon
```

Comment:
```suggestion
         {!cannotUseCustomDimensions &&<MantineIcon
```
so we don't display the alert if I can use custom dimensions

Reply:
Thanks, I missed this